docs/getting-started.md

```diff
@@ -0,0 +1,89 @@
+
+Price list assignment identifies the price list from which a product price is shown for customers. Moreover, price list assignments can do much more than simply provide default list and sale prices for products. This feature allows you to provide custom prices to targeted customers according to customer's characteristics such as age, gender, geographic location and searched terms.
+
+After this step completed, you can [Open new products in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#open-new-products-in-virto-commerce-storefront-frontend) , browse the catalog, add product to shopping cart and create order. You can view customer order via admin UI.
+
+## Step 4. Configure payments, shipments, taxes
+
+By default, you see demo payment and shipmen methods. Virto Commerce provides the ready to use module which. You can set your own configurations:
+
+* Configure [Payments](TODO) method
+* Configure [Shipments](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-shipments.md) method
+* Configure [Taxes](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-taxes.md) method
+
+## Step 5. Launch Marketing & promotional tools
+
+Marketing your store is essential to gain your customers loyalty and raise awareness among potential customers.
+
+There are several marketing tools in Virto Commerce: you can publish content, create personalized shopping experiences for customers, manage promotions.
+
+Start with Creation of two simple promotions:
+
+1. 5% if cart subtotal > 100 $
+2. Add Gif product
+
+Then you can play with extended e-commerce marketing, like 10% if product is in-stock.
+
+You can find more details here:
+
+* [Create a content publishing](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/marketing/how-to-add-an-advertising-spot-via-marketing/create-a-content-publishing.md)
+* [Dynamic content](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/marketing/dynamic-content.md)
+* [Managing promotions](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/marketing/promotions.md)
+* [Combine active promotions](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/marketing/combine-active-promotions.md)
+
+## Step 6. Configure User Experience
+
+To enhance User Experience Virto Commerce contains a set of tools:
+
+* [Creating a new menu item in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#creating-a-new-menu-item-in-virto-commerce-storefront-frontend)
```

Comment:
Review an add other activities which we already described in docs

Reply:
need your help to find activities

Reply:
fixed